src/context/auth-context.js

```diff
@@ -1,13 +1,13 @@
 import { useContext, createContext, useReducer, useEffect } from "react";
-import { Navigate } from "react-router-dom";
+import { useToast } from "./toast-context";
 import axios from "axios";
 import { userInitialState, authReducer } from "reducer/auth-reducer";
 
 const AuthContext = createContext();
 
 const AuthProvider = ({ children }) => {
   const [authState, authDispatch] = useReducer(authReducer, userInitialState);
-
+  const { setToastMsg, setToastState, setToastBg } = useToast();
   //  While login
   const loginHandler = async (e, formData) => {
```

Comment:
you have created this function as sync. But you are not calling any api's inside it. So please remove async

Reply:
I am calling api from line no 14 to 22 `/api/auth/login` at this end point